app/models/concerns/orangelight/document/export.rb

```diff
@@ -0,0 +1,38 @@
+module Orangelight
+  module Document
+    module Export
+      include ActiveSupport::Concern
+
+      # Override will_export_as method in Blacklight::Document::Export.
+      # Exclude certain export formats for records not in Voyager.
+      def will_export_as(short_name, content_type = nil)
+        return if formats_to_exclude.include? short_name
+        super
+      end
+
+      private
+
+        def formats
+          %i[marc marcxml refworks_marc_txt endnote openurl_ctx_kev]
+        end
+
+        def formats_to_exclude
+          return [] unless non_voyager?
+          formats
+        end
+
+        def holding_id
+          @holding_id ||= begin
+            holdings = JSON.parse(fetch(:holdings_1display, '{}')).first
+            holdings.blank? ? nil : holdings[0]
+          end
+        end
+
+        def non_voyager?
+          return true if %w[thesis visuals].include? holding_id
+          return true if fetch(:id, '').start_with?('SCSB')
```

Comment:
One small thing: can you do the SCSB check first (because there are tons more of those), and reverse the logic of this method?  I always find negative boolean methods confusing, so reversing the logic would let line 20 above be `return [] if voyager?` which seems clearer to me.

Reply:
Do we want to support marcxml exports for SCSB? Some of the citation scrapers like Zotero work better with Marc than what we are doing now. It would be easy enough to do if we were willing to create a field in the solr document that contained a blob of MARCXML since we do get MarcXML as input for those records.

Reply:
I think it's worth considering marcxml support for the SCSB records, though I don't want to hold up this ticket for that decision (which probably involves getting input from LISSC, etc.).  So I've created a ticket for that issue: https://github.com/pulibrary/orangelight/issues/1065

Reply:
Yeah, negative boolean method names are confusing. I waffled back and forth on that a bit. I'll change to the affirmative method name.